dashboard/app/views/layouts/application.html.haml

```diff
@@ -36,6 +36,15 @@
     - if current_user && current_user.age.nil?
       = render partial: 'layouts/age_interstitial'
 
+    #gdpr-dialog
+    - gdpr_data = {}
+    - if current_user
+      - already_accepted = current_user.properties[:data_transfer_agreement_accepted]
```

Comment:
Similar to above, we should be able to just use `current_user.data_transfer_agreement_accepted`.